backend/src/main/java/com/exadel/carinsurance/service/implementation/AuthService.java

```diff
@@ -75,9 +80,21 @@ public ResponseEntity signup( @RequestBody RegisterRequestEntity request ) {
   }
 
   @Override
-  public ResponseEntity login( @RequestBody AuthRequestEntity request ) {
+  public ResponseEntity login( @RequestBody AuthRequestEntity request,
+                               HttpServletRequest httpServletRequest,
+                               HttpServletResponse httpServletResponse ) {

```

Comment:
please don't inject raw `request`\`response` objects. It should be used only as a last resort or when you actually do a lot of operations with input\output properties.